StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/OnSetterFunctionCalledMessageHandler.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  OnSetterFunctionCalledMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/9/24.
+//
+
+import Foundation
+
+class OnSetterFunctionCalledMessageHandler<Values: Codable & Equatable>: ScriptMessageHandler<OnSetterFunctionCalledMessageHandler.Payload<Values>> {
+    struct Payload<Value: Codable & Equatable>: Codable {
+        let setter: String
+        let value: Value?
```

Comment:
nit: docstrings (originally https://github.com/stripe/stripe-ios/pull/3936#discussion_r1717578052)
```suggestion
        /// Name of the component-specific setter function (e.g. `onExit`)
        let setter: String
        /// Setter specific payload
        let value: Value?
```